rust/headless-client/src/lib.rs

```diff
@@ -73,14 +73,19 @@ struct Cli {
 impl Cli {
     fn command(&self) -> Cmd {
         // Needed for backwards compatibility with old Docker images
-        self.command.unwrap_or(Cmd::Standalone)
+        self.command.unwrap_or(Cmd::Auto)
     }
 }
 
 #[derive(clap::Subcommand, Clone, Copy)]
 enum Cmd {
+    /// If there is a token on disk, run in standalone mode. Otherwise, run as an IPC daemon
+    Auto,
     /// Listen for IPC connections and act as a privileged tunnel process for a GUI client
     Daemon,
+    /// Act as an IPC client for headless testing
+    #[command(hide = true)]
+    DebugIpcClient,
```

Comment:
I've seen applications that have a command caller `doctor` which performs some sanity checks. Perhaps we can borrow that idea here?

Reply:
hm it doesn't really do a health check, it's more like a stub IPC client until the GUI Client is performing the full IPC flow. Maybe `stub-ipc-client`?